UndertaleModTool/Scripts/Community Scripts/TouchControlsEnabler.csx

```diff
@@ -0,0 +1,156 @@
+var spr_controls_config = AddNewUndertaleSprite("spr_controls_config", 100, 13);
+var spr_button_scale = AddNewUndertaleSprite("spr_button_scale", 79, 9);
+var spr_analog_scale = AddNewUndertaleSprite("spr_analog_scale", 79, 12);
+var spr_analog_type = AddNewUndertaleSprite("spr_analog_type", 72, 12);
+var spr_reset_config = AddNewUndertaleSprite("spr_reset_config", 79, 12);
+var spr_controls_opacity = AddNewUndertaleSprite("spr_controls_opacity", 107, 13);
+var spr_arrow_leftright = AddNewUndertaleSprite("spr_arrow_leftright", 41, 9);
+var spr_black = AddNewUndertaleSprite("spr_black", 640, 480);
+
+spr_joybase.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_joybase1 });
+spr_joybase.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_joybase2 });
+spr_joystick.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_joystick });
+spr_joystick.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_joystick2 });
+spr_z_button.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_zbutton });
+spr_z_button.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_zbutton_p });
+spr_x_button.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_xbutton });
+spr_x_button.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_xbutton_p });
+spr_c_button.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_cbutton });
+spr_c_button.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_cbutton_p });
+spr_settings.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_settings_n });
+spr_settings.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_settings_p });
+spr_controls_config.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_controls_config });
+spr_button_scale.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_button_scale });
+spr_analog_scale.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_analog_scale });
+spr_analog_type.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_analog_type });
+spr_reset_config.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_reset_config });
+spr_controls_opacity.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_controls_opacity });
+spr_arrow_leftright.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_arrow_leftright });
+spr_black.Textures.Add(new UndertaleSprite.TextureEntry() { Texture = pg_black });
+
+Data.Sprites.Add(spr_joybase);
+Data.Sprites.Add(spr_joystick);
+Data.Sprites.Add(spr_z_button);
+Data.Sprites.Add(spr_x_button);
+Data.Sprites.Add(spr_c_button);
+Data.Sprites.Add(spr_settings);
+Data.Sprites.Add(spr_controls_config);
+Data.Sprites.Add(spr_button_scale);
+Data.Sprites.Add(spr_analog_scale);
+Data.Sprites.Add(spr_analog_type);
+Data.Sprites.Add(spr_reset_config);
+Data.Sprites.Add(spr_controls_opacity);
+Data.Sprites.Add(spr_arrow_leftright);
+Data.Sprites.Add(spr_black);
```

Comment:
*AFAIK* you should be able to use an `AddRange` function. à la `Data.Sprites.AddRange(spr_joybase, spr_joystick, spr_z_button)` etc.

Reply:
I changed the entire thing for adding sprites, it's inside a function now.